doc/tutorials/understanding-error-messages.md

```diff
@@ -0,0 +1,161 @@
+# Understanding Error Messages
+
+Errors can happen when trying to mount with insufficient permissions, missing plugin configurations or when you want to write
+specifications for your configurations and test it out or many other occasions. The error message gives information on what the problem is and possibly how
+to resolve them.
+
+Here is an example how you can trigger an error for an invalid IP entry in the `hosts` file:
+
+```sh
+sudo kdb mount --with-recommends /etc/hosts system/tests/hosts hosts
+sudo kdb set system/tests/hosts/ipv4/my 88.198.134.1777
+# RET: 5
+# Sorry, module network issued the error C03200:
+# Validation Semantic: name: system/tests/hosts/ipv4/my value: 88.198.134.1777 message: Name or service not known
+
+#cleanup
+sudo kdb umount system/tests/hosts
+```
+
+In this example we mount the hosts file under `/etc/hosts` into Elektra. Elektra also validates
+our input which is shown in the next line where we tried to set an invalid IP address. With this we can add or edit entries
+by using the tools of Elektra and do not need to edit the file directly.
+
+Note that you can always pass the **command line arguments -v (verbose) and -d (debug)** to the command line to get
```

Comment:
Imho it is not so important to make this bold. It is also a bit out of the blue that this is explained. You did not yet explain why `kdb set` above fails. Alternatively: explain why we would need `-v` or `-d` here.